src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityJdbcIT.java

```diff
@@ -8,7 +8,7 @@
 @Tag("virtual-schema")
 class RowLevelSecurityJdbcIT extends AbstractRowLevelSecurityIT {
     @Override
-    protected Map<String, String> getVirtualSchemaProperties() {
+    protected Map<String, String> getConnectionSpecificVirtualSchemaProperties() {
```

Comment:
I think this got wrong. This implementation should be in the `RowLevelSecurityLocalIT` class, and the one from `RowLevelSecurityLocalIT` should be here.

Reply:
Switched.